### ISSUE TITLE
Transition with scales instead of height

### ISSUE THREAD

**Issue by blasten** [PolymerElements/paper-toolbar#27]
From https://github.com/PolymerElements/paper-scroll-header-panel/issues/19

https://github.com/PolymerElements/paper-toolbar/blob/master/paper-toolbar.html#L94

**Comment by paulirish** [comment]
We're also encountering this perf issue on https://chromedevtools.github.io/debugger-protocol-viewer/#!/debugger-protocol-viewer/

I'd love to see the component drop height transitions because right now it means this animation is at 10fps which looks bad enough that we'd rather remove it.

**Comment by blasten** [comment]
@paulirish right, meanwhile you could use `paper-scroll-header-panel` that runs at 60fps.  :) https://github.com/PolymerElements/paper-scroll-header-panel

**Comment by notwaldorf** [comment]
I think this element will be deprecated in favour of the new `app-layout`, which is redesigned to be more efficient.
